composeApp/src/commonMain/kotlin/world/respect/app/view/learningunit/list/LearningUnitListScreen.kt

```diff
@@ -0,0 +1,187 @@
+package world.respect.app.view.learningunit.list
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.ListItem
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowCircleDown
+import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.ustadmobile.libuicompose.theme.black
+import org.jetbrains.compose.resources.stringResource
+import respect.composeapp.generated.resources.Res
+import respect.composeapp.generated.resources.clazz
+import respect.composeapp.generated.resources.duration
+import world.respect.app.app.RespectAsyncImage
+import world.respect.app.appstate.getTitle
+import world.respect.app.appstate.toDisplayString
+import world.respect.app.viewmodel.learningunit.list.LearningUnitListUiState
+import world.respect.app.viewmodel.learningunit.list.LearningUnitListViewModel
+import world.respect.datasource.opds.model.OpdsPublication
+
+@Composable
+fun LearningUnitListScreen(
+    viewModel: LearningUnitListViewModel
+) {
+    val uiState by viewModel.uiState.collectAsState()
+
+    LearningUnitListScreen(
+        uiState = uiState,
+        onClickLesson = { viewModel.onClickLesson(it) },
+        onClickFilter = { viewModel.onClickFilter(it) }
+    )
+}
+
+@Composable
+fun LearningUnitListScreen(
```

Comment:
This is not displaying the navigation - which makes it broken for any opds feed that uses navigation links (see index.json in the test cases).

Should be populated as follows

1) OpdsFeed.navigationLinks
2) OpdsFeed.publications
3) ForEach OpdsFeed.group - show header, then repeat 1 and 2